ui/src/app/service/workflow/workflow.store.ts

```diff
@@ -213,8 +213,13 @@ export class WorkflowStore {
             let workflowKey = key + '-' + detailedWorkflow.name;
             let store = this._workflows.getValue();
             let workflowToUpdate = store.get(workflowKey);
-            workflowToUpdate.groups = w.groups;
-            workflowToUpdate.last_modified = w.last_modified;
+            if (workflowToUpdate) {
+              workflowToUpdate.groups = w.groups;
+              workflowToUpdate.last_modified = w.last_modified;
+            } else {
```

Comment:
else must not happened.

Reply:
here https://github.com/ovh/cds/pull/2409